_includes/links.md

```diff
@@ -13,6 +13,7 @@
 [0202]: {{site.baseurl}}/recipe/0202-start-canvas/ "Load Manifest Beginning with a Specific Canvas"
 [0014]: {{site.baseurl}}/recipe/0014-accompanyingcanvas/ "Audio Presentation with Accompanying Image"
 [0015]: {{site.baseurl}}/recipe/0015-start/ "Begin playback at a specific point - Time-based media"
+[0139]: {{site.baseurl}}/recipe/0139-geo-annotation/ "Geolocate Canvas Fragment to a Point"
```

Comment:
I think this link should be: recipe/0139-geolocate-canvas-fragment/

Reply:
OOO man I see.  I thought this was going off the branch name, not the branch/recipe name.